tools/functional-tester/etcd-runner/main.go

```diff
@@ -310,10 +320,24 @@ func runWatcher(eps []string) {
 						return
 					}
 				}
-			}(prefixes[i])
+			}(roundPrefix + "-" + prefixes[i])
 		}
 	}
 	wg.Wait()
+
+	// cancel all watch channels
+	cancelc()
+
```

Comment:
@sinsharat BTW, the whole point of this test is to help us find bugs in client watcher code. so it is great if we can find bugs.